website/source/guides/operations/reference-architecture.html.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: "guides"
+page_title: "Nomad Reference Architecture"
+sidebar_current: "guides-operations-reference-architecture"
+description: |-
+  This document provides recommended practices and a reference
+  architecture for HashiCorp Nomad production deployments.
+product_version: 0.8
+---
+
+# Nomad Reference Architecture
+
+This document provides recommended practices and a reference architecture for HashiCorp Nomad production deployments. This reference architecture conveys a general architecture that should be adapted to accommodate the specific needs of each implementation.
+
+The following topics are addressed:
+
+- [Reference Architecture](#ra)
+- [Deployment Topology within One Datacenter](#one-dc)
```

Comment:
This would be less confusing if the sections were "Deployment Topology within a Single Region" and "Deployment Topology across Multiple Regions". Lets set the proper context in terms of Nomad's data model from the start.